spec/features/govuk_error_summary_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+RSpec.describe 'govuk_fieldset', type: :feature, js: true do
+  include_context 'components test page'
+  it 'exists' do
+    test_page.load
+    test_page.with_errors
+    expect(test_page).to have_govuk_error_summary
+  end
```

Comment:
This test needs at least 1 more test to prove that all of the errors are presented in the way GDS says they should be

Reply:
You can use 'elements' or 'sections' in site prism to represent a collection of things btw - might help to validate all the errors displayed